modules/manual-configuration-image-registry-ecr.adoc

```diff
@@ -22,9 +22,10 @@ Follow the instructions at https://docs.aws.amazon.com/AmazonECR/latest/userguid
 
 .Procedure
 . On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
-. Under the *Images* section, select *Google Container Registry*.
+. Under the *Image Integrations* section, select *Google Container Registry*.
```

Comment:
Should this step refer to Amazon Elastic Container Registry instead of Google Container Registry?